src/Altinn.Profile/Controllers/UserProfileInternalController.cs

```diff
@@ -93,7 +93,11 @@ public async Task<ActionResult<List<UserProfile>>> GetList([FromBody] List<Guid>
             return BadRequest();
         }
 
-        List<UserProfile> result = await _userProfileService.GetUserListByUuid(userUuidList);
-        return Ok(result);
+        Result<List<UserProfile>, bool> result = await _userProfileService.GetUserListByUuid(userUuidList);
+        List<UserProfile> userProfiles = result.Match(
+             userProfileList => userProfileList,
+             _ => []);
+             
+        return Ok(userProfiles);
```

Comment:
I must have been asleep at the wheel when we started using `Result<>` in Profile....

How did this come to your attention? Were you getting errors?

Reply:
I was trying to understand the patterns in the different classes, with a focus on the separation of concerns between Controller, (I)Service and (I)Client. That's when I noticed that `IUserProfileService.GetUserListByUuid` had a different signature than the other interface methods. 

As-is, the UserProfileService performs the resolving of the Result. My change in this PR is simply moving this step to the Controller classes

Reply:
I really really don't like how we're using a boolean to describe the failiure. Gives the controller nothing to work with. 

This is an example of correct use:
https://github.com/Altinn/altinn-notifications/blob/a58939e62d8b92653aa2c388b2b9a74a3ec7e501/src/Altinn.Notifications/Controllers/OrdersController.cs#L196

I'm not asking you to fix this now. Out of scope.